docs/articles/documentation/guides/basic-guides/select-page-elements.md

```diff
@@ -157,6 +157,33 @@ const sel = Selector('div').child();
 </html>
 ```
 
+The expression, associated with a selector is evaluated when this selector is used, not when it is declared. If page's content changes, subsequent selectors are reevaluated.
```

Comment:
> If page's content changes, subsequent selectors are reevaluated.

It's not true. 

`If page's content changes, the selector can point to another element or there is no element satisfied the selector`.

Reply:
edited